collectors/apps.plugin/README.md

```diff
@@ -64,6 +64,23 @@ Each of these sections provides the same number of charts:
     -   Major Page Faults (i.e. swap activity)
 -   Network
     -   Sockets Open
+-   eBPF syscall    
```

Comment:
I do think @ilyam8 is right that we should clarify these charts come from another place, so let's add this one more line and then I think we're done!

```suggestion
In addition, if the [eBPF collector](/collectors/ebpf.plugin/README.md) is running, your dashboard will also show the following charts using low-level Linux metrics:

-   eBPF syscall    
```

Reply:
I wrote it as you requested.